services/viva-ms/lambdas/syncOfficers.js

```diff
@@ -0,0 +1,160 @@
+/* eslint-disable no-console */
+import AWS from 'aws-sdk';
+import to from 'await-to-js';
+import deepEqual from 'deep-equal';
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+
+import config from '../../../config';
+import params from '../../../libs/params';
+import hash from '../../../libs/helperHashEncode';
+import * as request from '../../../libs/request';
+import * as dynamoDb from '../../../libs/dynamoDb';
+
+const SSMParams = params.read(config.vada.envsKeyName);
+
+const dynamoDbConverter = AWS.DynamoDB.Converter;
+
+/**
+ * Handler reacting on case event
```

Comment:
I would like a better function description, what's the reason we react on a case event?  To do what?